mis_builder/tests/test_aep.py

```diff
@@ -363,8 +367,17 @@ def test_inactive_tax(self):
         self.aep.parse_expr(expr)
         self.aep.done_parsing()
 
+        tax_group = self.env["account.tax.group"].create(dict(name="test tax group"))
+
         tax = self.env["account.tax"].create(
-            dict(name="test tax", active=True, amount=0, company_id=self.company.id)
+            dict(
+                name="test tax",
+                active=True,
+                amount=0,
+                company_id=self.company.id,
+                tax_group_id=tax_group.id,
+                country_id=self.company.country_id.id,
```

Comment:
```suggestion
                country_id=self.company.country_id.id,
```

We don't need to set the country here if it is set on company_id.